packages/Panel/stories/examples/CoPlanarPanel.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Panel from "../../src";
+import { TextLine } from "../helpers";
+
+export default function CoPlanarPanel() {
+  const [isOpen, setIsOpen] = React.useState(true);
+  const toggle = () => {
+    setIsOpen(state => !state);
+  };
+
+  return (
+    <>
+      <Panel isOpen={isOpen} onClose={toggle} offset={{ top: 50 }}>
+        <Panel.Header isSticky>Header isSticky</Panel.Header>
```

Comment:
I think we should also include `hasAccent` – unless I'm mistaken, that's the pattern in the new [design guideline](https://aclgrc.atlassian.net/wiki/spaces/UX/pages/2014052656/Panels+Approved).